edb/schema/operators.py

```diff
@@ -60,6 +60,9 @@ class Operator(s_func.CallableObject, s_abc.Operator):
     commutator = so.SchemaField(
         so.Object, default=None, compcoef=0.99)
 
+    recursive = so.SchemaField(
```

Comment:
Make sure you read and populate the field in `intromech.read_operators`

Reply:
Done.

Reply:
Not quite, you also need to update `pgsql/datasources/schema/operators.py`

Reply:
OK, added `0.recursive` to the query there.